core/src/components/card/card.tsx

```diff
@@ -7,7 +7,7 @@ import { generateUniqueId, hasSlot } from '../../utils/utils';
  * @slot thumbnail - Slot for the Card thumbnail.
  * @slot body - Slot for the body section of the Card.
  * @slot body-image - Slot for the body section of the Card, used for image.
- * @slot bottom - Slot for the bottom section of the Card.
```

Comment:
Do we want to update the slot name for the icon in Storybook? 
![image](https://github.com/scania-digital-design-system/tegel/assets/48509966/bda2c0d3-40e5-4aed-82e5-27996d73fa15)